terraform/modules/emrfs-lambda/policy-munge-lambda-files/tests/lambda_handler_tests.py

```diff
@@ -212,22 +213,17 @@ def test_tag_role_with_policies_NOT_CHUNKED(self, mock_tag_role):
     def test_tag_role_with_policies_CHUNKED(self, mock_tag_role):
         lambda_handler.tag_role_with_policies(['policy_one', 'policy_two'], 'emrfs_user_one', variables['common_tags'])
 
-        mock_tag_role.assert_called_with('emrfs_user_one', [
-            {
-                'Key': 'tag1key',
-                'Value': 'tag1val'
-            },
-            {
-                'Key': 'tag2key',
-                'Value': 'tag2val'
-            },
-            {
-                'Key': 'InputPolicies-1of2',
-                'Value': 'policy_one'
-            },
-            {
-                'Key': 'InputPolicies-2of2',
-                'Value': 'policy_two'
-            },
-        ])
+        mock_tag_role.assert_called_with(
+            'emrfs_user_one', [
+                {'Key': 'tag1key', 'Value': 'tag1val'},
+                {'Key': 'tag2key', 'Value': 'tag2val'},
+                {'Key': 'InputPolicies-1of2', 'Value': 'policy_one'},
+                {'Key': 'InputPolicies-2of2', 'Value': 'policy_two'},
+            ]
+        )
 
+    def test_verify_policies(self):
+        with self.assertRaises(NameError):
+            lambda_handler.verify_policies(['policy_two'], mocked_policy_object_list)
+        self.assertIsNone(lambda_handler.verify_policies(['policy_one', 'policy_three'], mocked_policy_object_list))
+        self.assertIsNone(lambda_handler.verify_policies(['policy_one'], mocked_policy_object_list))
```

Comment:
This is three tests. They should really be split in to three test methods with the name of the method describing the test. I.e. `test_verify_policies_raises_error_when_policy_not_matched` or similar and so on.